pkg/kubectl/testing/types.generated.go

```diff
@@ -158,7 +158,13 @@ func (x *TestStruct) CodecEncodeSelf(e *codec1978.Encoder) {
 				z.EncSendContainerState(codecSelfer_containerArrayElem1234)
 				if yyq2[2] {
 					yy10 := &x.ObjectMeta
-					yy10.CodecEncodeSelf(e)
+					yym11 := z.EncBinary()
```

Comment:
So I ran hack/update-all.sh again, which runs hack/update-codecgen and I don't get any more diffs. I am not entirely sure what is going on.